app/presenters/menu/section.rb

```diff
@@ -84,5 +94,9 @@ def parent_path(acc = [])
       acc << id
       @parent.present? ? @parent.parent_path(acc) : acc
     end
+
+    def load_children?
```

Comment:
This is related to TreeBuilder changes below. Makes Menu::Section structs play nice with TreeBuilder.